Sources/tbDEX/HttpClient/tbDEXHttpClient.swift

```diff
@@ -61,13 +91,16 @@ public enum tbDEXHttpClient {
         }
 
         var request = URLRequest(url: url)
```

Comment:
There's also a URL change in this spec change. for rfq, it's `/exchange` and `/exchange/:exchangeId` for order and close.